frontend/src/components/Admin/AdminReview.tsx

```diff
@@ -172,6 +179,18 @@ const AdminReviewComponent = ({ review, setToggle, declinedSection }: Props): Re
                 <Typography className={dateText}>{formattedDate}</Typography>
               </Grid>
 
+              <Grid item xs={12} className={bedroomsPriceText}>
+                {bedrooms > 0 && (
+                  <Typography style={{ fontWeight: '600' }}>Bedroom(s): {bedrooms}</Typography>
+                )}
+                {price > 0 && (
+                  <Typography style={{ fontWeight: '600' }}>
+                    {' '}
```

Comment:
This is a smart way to add padding, but doesn't account for the case where there is price information but no bedroom information. Perhaps have an additional check for whether bedrooms > 0 before adding the tab.